lib/config/migrate-validate.ts

```diff
@@ -41,6 +42,9 @@ export async function migrateAndValidate(
       // TODO #556 - enable warnings in real PRs
       massagedConfig.warnings = (config.warnings || []).concat(warnings);
     }
+    if (!input.privateKey && input.privateKeyPath) {
+      massagedConfig.privateKey = fs.readFileSync(config.privateKeyPath);
```

Comment:
I think this is not the right logical place for it to be read, as it will be read once for every repo. Instead, better to do it as part of global config loading.

Reply:
Thanks for your Feedback. Moved to `lib/config/index.js`.